Tests/SwiftParserTest/translated/GuardTests.swift

```diff
@@ -24,7 +24,13 @@ final class GuardTests: XCTestCase {
       """,
       diagnostics: [
         DiagnosticSpec(message: "expected 'else' and body in 'guard' statement", fixIts: ["insert 'else' and body"])
-      ]
+      ],
+      fixedSource: """
+        func noConditionNoElse() {
+          guard {} else {
+        }
```

Comment:
The closing brace should be indented. This will probably be a bigger issue to fix but a one worth filing.

Reply:
https://github.com/apple/swift-syntax/issues/1601